cmd/kubeadm/app/util/apiclient/idempotency.go

```diff
@@ -186,3 +192,49 @@ func CreateOrUpdateClusterRoleBinding(client clientset.Interface, clusterRoleBin
 	}
 	return nil
 }
+
+// PatchNode tries to patch a node using the following client, executing patchFn for the actual mutating logic
+func PatchNode(client clientset.Interface, nodeName string, patchFn func(*v1.Node)) error {
```

Comment:
Do you patch b/c dynamic kubelet config is not default?

Reply:
No this is used when e.g. setting the master label/taint, and generically modifying the Node object.